install.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash 
+
+git clone https://github.com/systers/mentorship-backend.git
+cd mentorship-backend
+rm .env
+echo "Setup: please enter details"
+
+echo "FLASK_ENVIRONMENT_CONFIG=<dev-or-test-or-prod>" && read FLASK_ENVIRONMENT_CONFIG
+if [[ -z "$FLASK_ENVIRONMENT_CONFIG" ]]; then
+   printf '%s\n' "No input entered"
+   exit 1
+fi
+echo "SECRET_KEY=<your-secret-key>" && read SECRET_KEY
+if [[ -z "$SECRET_KEY" ]]; then
+   printf '%s\n' "No input entered"
+   exit 1
+fi
+echo "SECURITY_PASSWORD_SALT=<your-security-password-salt>" && read SECURITY_PASSWORD_SALT
+if [[ -z "$SECURITY_PASSWORD_SALT" ]]; then
+   printf '%s\n' "No input entered"
+   exit 1
+fi
+echo "MAIL_DEFAULT_SENDER=<mail-default-sender>" && read MAIL_DEFAULT_SENDER
+if [[ -z "$MAIL_DEFAULT_SENDER" ]]; then
+   printf '%s\n' "No input entered"
+   exit 1
+fi
+echo "MAIL_SERVER=<mail-server>" && read MAIL_SERVER
+if [[ -z "$MAIL_SERVER" ]]; then
+   printf '%s\n' "No input entered"
+   exit 1
+fi
+echo "APP_MAIL_USERNAME=<app-mail-username>" && read APP_MAIL_USERNAME
+if [[ -z "$APP_MAIL_USERNAME" ]]; then
+   printf '%s\n' "No input entered"
+   exit 1
+fi
+echo "APP_MAIL_PASSWORD=<app-mail-password>" && read APP_MAIL_PASSWORD
+if [[ -z "$APP_MAIL_PASSWORD" ]]; then
+   printf '%s\n' "No input entered"
+   exit 1
+fi
+
+echo "export FLASK_ENVIRONMENT_CONFIG=$FLASK_ENVIRONMENT_CONFIG" >> .env
+echo "export SECRET_KEY=$SECRET_KEY" >> .env
+echo "export SECURITY_PASSWORD_SALT=$SECURITY_PASSWORD_SALT" >> .env
+echo "export MAIL_DEFAULT_SENDER=$MAIL_DEFAULT_SENDER" >> .env
+echo "export MAIL_SERVER=$MAIL_SERVER" >> .env
+echo "export APP_MAIL_USERNAME=$APP_MAIL_USERNAME" >> .env
+echo "export APP_MAIL_PASSWORD=$APP_MAIL_PASSWORD" >> .env
+
+# create venv
+python3 -m venv venv
```

Comment:
Not everyone will use virtualenv via this command.
A better way is the step 1 mentioned in our `README`
https://github.com/systers/mentorship-backend#run-app
I think we should do something like
`pip3 install --user virtualenv`
and then `virtualenv --python /usr/bin/python3.6 venv`

Reply:
I ran into the same trouble while installing it.

Reply:
Done please review